src/main/java/com/ecore/roles/web/MembershipsApi.java

```diff
@@ -0,0 +1,17 @@
+package com.ecore.roles.web;
+
+import com.ecore.roles.web.dto.MembershipDto;
+import org.springframework.http.ResponseEntity;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface MembershipsApi {
+
+    ResponseEntity<MembershipDto> assignRoleToMembership(
```

Comment:
Please follow the methods names rules mentioned above in the service layer of this domain.